web/styles/popovers.css

```diff
@@ -1074,6 +1035,85 @@ ul {
     }
 }
 
+.popover-menu-user-header {
```

Comment:
Created new class names instead of using the ones previously being used in
the personal settings popover with the intention of eventually replacing
the classes used in the personal settings popover with the CSS here.

Reply:
Sounds good, but maybe we should spell it `popover-menu-user-card-header`